src/api/app/jobs/send_event_emails_job.rb

```diff
@@ -39,29 +39,28 @@ def create_rss_notifications(event)
   end
 
   def notification_dynamic_params(event)
-    if event.eventtype == 'RequestStatechange'
-      return {
-        notifiable_type: 'BsRequest',
-        notifiable_id: event.payload['id'],
-        bs_request_state: event.payload['state'],
-        bs_request_oldstate: event.payload['oldstate']
-      }
-    end
+    return {} unless event.eventtype.in?(['Event::RequestStatechange',
+                                          'Event::RequestCreate',
```

Comment:
You can extract this array into a constant. Maybe something like `EVENTS_TO_NOTIFY`

Reply:
Done. Please re-review.